zaza/openstack/charm_tests/mysql/tests.py

```diff
@@ -1123,7 +1123,7 @@ def setUpClass(cls, application_name="keystone-mysql-router"):
         cls.services = ["mysqlrouter"]
         # Config file affected by juju set config change
         cls.conf_file = (
-            "/var/lib/mysql/{}-mysql-router/mysqlrouter.conf"
+            "/var/lib/mysql/{}/mysqlrouter.conf"
```

Comment:
This change looks curious?  i.e. the router directory is changing from `/var/lib/mysql/keystone-mysql-router/mysqlrouter.con` to `/var/lib/mysql/keystone/mysqlrouter.conf` but I can't find a corresponding change in https://review.opendev.org/c/openstack/charm-mysql-router/+/886810

Please could you help me out, and explain this change?

Reply:
oh sorry about this, 
I meant to ask this.. 
While I'm testing zaza test, self.conf_file returned keystone-mysql-router-mysql-router as application name was keystone-mysql-router.

I think it is wrong in the beginning. but I'm not sure 100%.

Reply:
> This change looks curious? i.e. the router directory is changing from `/var/lib/mysql/keystone-mysql-router/mysqlrouter.con` to `/var/lib/mysql/keystone/mysqlrouter.conf` but I can't find a corresponding change in https://review.opendev.org/c/openstack/charm-mysql-router/+/886810
> 
> Please could you help me out, and explain this change?

Could you please check my comment if it makes sense of I need to change it to original?

Thanks in advance.

Reply:
I'm not clear on *why* you've made this change.  Did the test break?  Also, the associated gerrit review also needs some changes please.  Thanks.

Reply:
> I'm not clear on _why_ you've made this change. Did the test break? Also, the associated gerrit review also needs some changes please. Thanks.

Yes. my test was failed because self.conf_file returned keystone-mysql-router-mysql-router instead of keystone-mysql-router as I mentioned earlier. Thanks!

Reply:
fixing tests against to jammy as well

Reply:
> I'm not clear on _why_ you've made this change. Did the test break? Also, the associated gerrit review also needs some changes please. Thanks.

I mentioned why I changed this one. Could you please review this again? Thanks in advance!

Reply:
I'll quote myself:

> This change looks curious? i.e. the router directory is changing from /var/lib/mysql/keystone-mysql-router/mysqlrouter.con to /var/lib/mysql/keystone/mysqlrouter.conf but I can't find a corresponding change in https://review.opendev.org/c/openstack/charm-mysql-router/+/886810

> Please could you help me out, and explain this change?

I'm confused.  either `test_910_restart_on_config_change` used to break silently or something else has changed, but I can't work out what is different.  If you can't explain *why* this is different then I can't land the change without personally setting it up and running it to see *why* this is different for this patch.  i.e. before this patch the test 'passed' (but may not have been working??), and it also passes with this change, which makes no sense, as one version should break as there are no changes in the associated mysql-router charm patch on gerrit to affect this.

Reply:
I removed the change and I put seperated one into specific test
I hope this is proper change.
Thanks.